internal/mode/static/state/graph/gateway_test.go

```diff
@@ -870,6 +913,77 @@ func TestBuildGateway(t *testing.T) {
 			},
 			name: "nil gatewayclass",
 		},
+		{
+			gateway: createGateway(
+				gatewayCfg{listeners: []v1.Listener{foo443TLSListener, foo443HTTPListener}},
+			),
+			gatewayClass: validGC,
+			expected: &Gateway{
+				Source: getLastCreatedGetaway(),
```

Comment:
Just noticed the spelling of this function is wrong. Can we fix that? `getLastCreatedGateway`